ansible/inventory/env/group_vars/all.yml

```diff
@@ -660,3 +660,6 @@ dp_postgres_username: analytics
 
 # Will enable cassandra cluster if number of cassandra nodes > 1
 cassandra_cluster_size: "{{ groups['cassandra'] | length }}"
+
+# Azure sendgrid mail server apitoken username
+mail_server_username: "apikey"
```

Comment:
Why are we defining a default here?

Reply:
If its only the username to avoid template errors, we should have password and host fields also.

Reply:
this is a constant for sendgrid. That'll be for the default installations. If you want to override you can do it in common.yaml

Reply:
Okay, please update the comment for this as default value used in case of sendgrid

Reply:
sure